sdk/core/azure-core/src/main/java/com/azure/core/credential/TokenCredential.java

```diff
@@ -16,4 +16,13 @@
      * @return a Publisher that emits a single access token
      */
     Mono<AccessToken> getToken(TokenRequestContext request);
+
+    /**
+     * The options to configure the token refresh behavior.
+     *
+     * @return the current offset for token refresh
+     */
+    default TokenRefreshOptions getTokenRefreshOptions() {
```

Comment:
Update the changelog in azure-core to log this new API.

Reply:
updated